### PR TITLE
Enable power management cpufreq/cpufreq_governor test in kpet-db

### PR DESCRIPTION
**Pull request by idorax**

### CODE REVIEW

**Comment by idorax** [comment]
A manual test for RHEL8 is done, please refer to:
- http://pastebin.test.redhat.com/811267

File: power-management/cpufreq/cpufreq_governor/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+FILE=$(readlink -f ${BASH_SOURCE})
+NAME=$(basename $FILE)
```
Comment (emanchado) [inline power-management/cpufreq/cpufreq_governor/runtest.sh]: I noticed this variable doesn't seem to be used for anything. Is it a convention to always have it there? If not, maybe remove it?

Comment (idorax) [reply power-management/cpufreq/cpufreq_governor/runtest.sh]: Well, those useless "NAME" and "TEST" are removed from all `runtest.sh` in directory **power-management**. Please take a look again, thanks!

File: power-management/cpufreq/cpufreq_governor/Makefile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+clean:
+	rm -f *~ $(BUILT_FILES)
+
+include /usr/share/rhts/lib/rhts-make.include
+
+$(METADATA): Makefile
+	@echo "Owner:        Erik Hamera <ehamera@redhat.com>" > $(METADATA)
+	@echo "Name:         $(TEST)" >> $(METADATA)
+	@echo "TestVersion:  $(TESTVERSION)" >> $(METADATA)
+	@echo "Path:         $(TEST_DIR)" >> $(METADATA)
```
Comment (emanchado) [inline power-management/cpufreq/cpufreq_governor/Makefile]: Wait, hm, where is `TEST_DIR` defined?

Comment (idorax) [reply power-management/cpufreq/cpufreq_governor/Makefile]: `TEST_DIR` is defined by file `/usr/share/rhts/lib/rhts-make.include`, 
e.g. 
```bash
$ egrep TEST_DIR= /usr/share/rhts/lib/rhts-make.include
TEST_DIR=/mnt/tests$(TEST)
```

**Pull request merged**